src/db/index.js

```diff
@@ -37,14 +41,18 @@ const initDB = async () => {
       db.Withdraws.loadDatabase(),
       db.Blocks.loadDatabase(),
       db.TransactionReceipts.loadDatabase(),
+      db.GlobalLeaderboard.loadDatabase(),
+      db.EventLeaderboard.loadDatabase(),
     ]);
 
     await db.Blocks.ensureIndex({ fieldName: 'blockNum', unique: true });
     await db.Events.ensureIndex({ fieldName: 'txid', unique: true });
     await db.Bets.ensureIndex({ fieldName: 'txid', unique: true });
     await db.ResultSets.ensureIndex({ fieldName: 'txid', unique: true });
     await db.Withdraws.ensureIndex({ fieldName: 'txid', unique: true });
-    await db.TransactionReceipts.ensureIndex({ fieldName: 'transactionHash', unique: true });
+    // await db.TransactionReceipts.ensureIndex({ fieldName: 'transactionHash', unique: true });
```

Comment:
Undo this. I synced from scratch today and did not run into this problem. Chalking it up to bad data.